### DISCUSSION TITLE
Question about Kyty and Obliteration

### DISCUSSION THREAD

**Discussion by SoffiMeower** [obhq/obliteration#711] [Q&A]
From what I've seen, you and InoriRus have been working together on both Kyty and Obliteration, are they at one point going to be merged together for ease of development? Plus it'd help reduce user confusion.

**Comment by ultimaweapon** [comment]
> you and InoriRus have been working together on both Kyty and Obliteration,

No. For Kyty I just created a PR to add GitHub Actions. For Obliteration InoriRus does not involved in the development anyway. The reason GitHub show InoriRus name in the Contributors because this repository contains commits from Kyty.

So it is completely separated project and I think it will never merged together. I'll update README to make it more clear.

**Comment by SoffiMeower** [comment]
...I wonder which one will prevail .w.

**Comment by ultimaweapon** [comment]
Just one of it is enough like PS2 we have PCSX2 and PS3 we have RPCS3. So let's hope we got one for PS4 too :)

**Comment by SuchAFuriousDeath** [comment]
> ...I wonder which one will prevail .w.
We know now :D
